src/status_im2/common/theme/core.cljs

```diff
@@ -31,5 +30,4 @@
 
 (defn set-theme
   [value]
-  (quo/set-theme value)
```

Comment:
@Parveshdhull, it appears that this was introduced during the [Refactor app theme management](https://github.com/status-im/status-mobile/pull/15455/files#diff-171c55d08d05cd1b3a6f75b068b1585a549769cc452d02b4c00a1556d351b3c8R23-R24) process. I am in the process of removing it as part of the refactoring. I wanted to inform you of this change and ensure that it does not impact any essential functionality.

Reply:
We need to put this `quo/set-theme` usage back to fix the issue shared by @VolodLytvynenko here: https://github.com/status-im/status-mobile/pull/17404#issuecomment-1735060373. Otherwise, the old UI elements will be broken until we fully migrate everything.

I can put this back and exclude this file from the linting script for now and create a follow-up issue to remove this usage later. @flexsurfer What do you suggest?

Reply:
let's do the hack with the comment for now, just set quo/set-theme to the atom and use atom here, so we could keep quo linting error, so nobody uses quo by mistake

Reply:
yeah i was wondering why you didn't remove quo.theme then, so it's still used

Reply:
@flexsurfer correct me if I'm wrong, but from my understanding, we can't start removing code until we remove all status_im usage from status_im2.

I didn't get what you meant by 'set quo/set-theme to the atom and use atom here'. Could you please elaborate?